src/ppx_deriving_qcheck/ppx_deriving_qcheck.ml

```diff
@@ -1,243 +1,232 @@
 open Ppxlib
+module G = Qcheck_generators
```

Comment:
Perhaps use `QCheck_generators` for for consistency with `QCheck`?

Reply:
Also done in 9d846dfe0a8d2032212c01f66bdb65b8009ec534